src/js/components/AccordionPanel/doc.js

```diff
@@ -31,6 +31,11 @@ export const themeDoc = {
     type: 'React.Element',
     defaultValue: '<FormDown />',
   },
+  'accordion.icons.color': {
+    description: 'The icon color to use in the accordion.',
+    type: 'string',
```

Comment:
```suggestion
    type: 'string | { dark: string, light: string }',
```
Grommet's `color` supports both string and a dark/light object, see example here https://codesandbox.io/s/accordion-custom-icon-wxr4n

Reply:
I'll update this